feat(changelog): iam-added-scim-user-groups-are-now-supported 2026-08-18 (#6804)
scaleway/docs-content: changelog/august2026/2026-08-18-iam-added-scim-user-groups-are-now-supported.mdx

---
title: SCIM - User groups are now supported
status: added
date: 2026-08-18
category: security-identity
product: iam
---

> SCIM user provisioning now support user groups.

Reviewer: ```suggestion
SCIM user provisioning now supports user groups.
```

> If you use Entra ID, enable group provisioning from `Attribute Mapping` in the `Manage` menu. Select `Provision Microsoft Entra ID Groups`, then set `Enabled` to `Yes`.

Reviewer 2: ```suggestion
If you use Entra ID, enable group provisioning from **Attribute Mapping** in the **Manage** menu. Select **Provision Microsoft Entra ID Groups**, then set **Enabled** to **Yes**.
```

Reviewer 2: If this is a GUI-based procedure rather than a CLI-based one, then I suggest the above changes.

> Note 1 :  If you are on Entra ID, activate groupe provisioning by going to `Attribute Mapping` in the `Manage` menu, select `Provision Microsoft Entra ID Groups`, and select `Yes` on `Enabled`.

Reviewer: ```suggestion
If you use Entra ID, enable group provisioning from `Attribute Mapping` in the `Manage` menu. Select `Provision Microsoft Entra ID Groups`, then set `Enabled` to `Yes`.
```

> Note 2 : Nested groups aren't supported.

Reviewer: ```suggestion
Nested groups are not supported.
```
